src/mmw/apps/user/views.py

```diff
@@ -81,10 +82,14 @@ def logout(request):
         return redirect('/')
 
 itsi = ItsiService()
+ITSI_URL_FLAG = settings.ITSI['url_flag']
 
 
 def itsi_login(request):
-    redirect_uri = request.build_absolute_uri(reverse('itsi_auth'))
+    redirect_uri = '{0}?next={1}'.format(
```

Comment:
I need `.format` for auto parsing keys from a dictionary here: https://github.com/WikiWatershed/model-my-watershed/pull/578/files#diff-76f6d7d05a4b988f9b9af9034c53a081R119 so I decided to be consistent and just use it everywhere. Reading up on recent Python guidelines, it seems like there is a gradual movement away from `%`